src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -40,6 +40,12 @@ namespace Microsoft.CodeAnalysis.CSharp.Semantic.UnitTests.Semantics
 
     // PROTOTYPE(semi-auto-props): Add ENC tests.
 
+    // PROTOTYPE(semi-auto-props): Add a test where a virtual `get; set;` auto property is overridden by a sealed `get => 0;`.
+    // A synthesized sealed accessor is expected to be produced.
+
+    // PROTOTYPE(semi-auto-props): Add a test where a virtual property is `public virtual int P6 { get => 0; set { } }`, and
+    // it's overridden by `public override int P6 { get => field; }`
+    // The assignment of the property in constructor should use the base setter.
```

Comment:
This won't be allowed per https://github.com/dotnet/csharplang/blob/main/meetings/2022/LDM-2022-05-02.md#partial-overrides-of-virtual-properties